src/test/compile-fail/unsized-locals/by-value-trait-object-safety.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
```

Comment:
remove the copyright header and move the test to `src/test/ui`

Reply:
Sure, done.